docs/machineset-azure.md

```diff
@@ -9,20 +17,21 @@ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 
 `<zone>` should be replaced with a valid Azure availability zone like `us-east-1a`.
 
-`<image>` should be a WindowsServer image offering that defines the 2019-Datacenter-with-Containers SKU with version
-          17763.1457.2009030514 or earlier. Run the following command to list Azure image info:
+`<image>` <image> should be a WindowsServer image offering that defines the 2019-Datacenter-with-Containers SKU
+
+Run the following command to list compatible Azure images:
 ```shell script
   az vm image list --all --location <location> \
       --publisher MicrosoftWindowsServer \
       --offer WindowsServer \
-      --sku 2019-Datacenter-with-Containers \
-      --query "[?contains(version, '17763.1457.2009030514')]"
+      --sku 2019-Datacenter-with-Containers
```

Comment:
Did you run this command and verify that it does what it says? Moreover we only added it because of the version, so I suggest removing it.

Reply:
> Did you run this command and verify that it does what it says?

Yes

>Moreover we only added it because of the version, so I suggest removing it.

Doesn't hurt, it helps to find the correct `<image>` value

Reply:
Then please add it for AWS also.